docs/source/customization.rst

```diff
@@ -10,10 +10,20 @@ Customization and extension of functionality
 DataLad provides numerous commands that cover many use cases. However, there
 will always be a demand for further customization or extensions of built-in
 functionality at a particular site, or for an individual user. DataLad
-addresses this need by providing a generic plugin interface.
+addresses this need with two mechanisms:
 
-Using plugins
-=============
+- Plugins_
+- `Extension packages`_
```

Comment:
see https://github.com/datalad/datalad-module-template/issues/3

Reply:
Address in that repo.